pkg/startup/startup.go

```diff
@@ -421,19 +421,24 @@ func configureIPsAndSubnets(node *api.Node) (bool, error) {
 			log.Warnf("Autodetection of IPv6 address failed, keeping existing value: %s", node.Spec.BGP.IPv6Address)
 			validateIP(node.Spec.BGP.IPv6Address)
 		}
-	} else {
+	} else if ipv6Env != "none" {
 		if ipv6Env != "" {
 			node.Spec.BGP.IPv6Address = parseIPEnvironment("IP6", ipv6Env, 6)
 		}
 		validateIP(node.Spec.BGP.IPv6Address)
 	}
 
+	if ipv4Env == "none" && (ipv6Env == "" || ipv6Env == "none") {
+		log.Warn("No IP Addresses configured or set to autodetect")
```

Comment:
"No IP addresses configured, and autodetection is not enabled" is probably more clear / correct.